.github/workflows/ci_cd_pr.yml

```
@@ -233,3 +233,12 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           auditing-level: 'high'
           trust-ansys-actions: true
+  build-library:
+    name: "Build library basic example"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Build library source and wheel artifacts"
+        uses: ansys/actions/build-library@fix/unify-artifacts-file-names
+        with:
+          library-name: ${{ env.LIBRARY_NAME }}
+          working-directory: ".ci"
```

This expects the path to the directory containing the actual project file. Thus:

```suggestion
          working-directory: ".ci/ansys-actions"
```

---

Not needed anymore. Testing done in pymapdl repository.